Tasks/VsTest/vstest.ts

```diff
@@ -699,6 +699,11 @@ function invokeVSTest(testResultsDirectory: string): Q.Promise<number> {
                 .then(function (vsVersion) {
                     try {
                         vstestLocation = getVSTestLocation(vsVersion);
+                        vsTestVersionForTIA = getVsTestVersion();
```

Comment:
we have to call getVsTestVersion (even if tia is disabled) because it populates vstestversionfortia array and we are using that to decide if /diag argument should be pushed or not